app/src/main/java/ch/epfl/qedit/view/QuizActivity.java

```diff
@@ -23,7 +23,7 @@ protected void onCreate(Bundle savedInstanceState) {
         // Pass the first question as argument to the new QuestionFragment
         Bundle bundle = new Bundle();
         bundle.putSerializable("q0", question);
-        QuestionFragment frag = QuestionFragment.newInstance();
+        QuestionFragment frag = new QuestionFragment();
```

Comment:
The newInstance() seems to be a common pattern especially when creating classes with Android Studio. But it is indeed not very useful ...